_i18n/it/get-started/faq.md

```diff
@@ -1,136 +1,142 @@
 
 </div>
 
 </div>
 
 <div class="tab">
     <input id="tab-two" type="checkbox" name="tabs" class="accordion">
-    <label for="tab-two" class="accordion">How can I get Monero?</label>
+    <label for="tab-two" class="accordion">Come posso ottenere Monero?</label>
     
 <div class="tab-content" markdown="1">
 
-You can buy Monero from an exchange or from an individual. Alternatively, you can try mining Monero to get coins from the block reward.
+Puoi acquistare Monero da un "exchange" (cambiavalute) o da un privato.
+Alternativamente puoi provare a minare Monero per ottenere monete dalla ricompensa del blocco.
 </div>
 
 </div>
 
 <div class="tab">
     <input id="tab-three" type="checkbox" name="tabs" class="accordion">
-    <label for="tab-three" class="accordion">What is a mnemonic seed?</label>
+    <label for="tab-three" class="accordion">Cos'è un seed mnemonico?</label>
     
 <div class="tab-content" markdown="1">
 
-A mnemonic seed is a set of 25 words that can be used to restore your account anywhere. Keep these words safe and do not share them with someone else. You can use this seed to restore your account, even if your computer crashes.
+Un seed mnemonico (mnemonic seed) è una sequenza di 25 parole che può essere utilizzata per recuperare il tuo conto dovunque tu voglia. Tieni queste parole al sicuro e non condividerle con nessuno. Puoi usare questo seed per recupeare il tuo conto, anche se il tuo computer si rompe.
 </div>
 
 </div>
 
 <div class="tab">
     <input id="tab-four" type="checkbox" name="tabs" class="accordion">
-    <label for="tab-four" class="accordion">How is Monero’s privacy different from other coins?</label>
+    <label for="tab-four" class="accordion">In cosa è diversa la privacy offerta da Monero rispetto a quella offerta dalle altre criptovalute?</label>
     
 <div class="tab-content" markdown="1">
 
-Monero uses three different privacy technologies: ring signatures, ring confidential transactions (RingCT), and stealth addresses. These hide the sender, amount, and receiver in the transaction, respectively. All transactions on the network are private by mandate; there is no way to accidentally send a transparent transaction. This feature is exclusive to Monero. You do not need to trust anyone else with your privacy.
+Monero utilizza tre differenti tecnologie mirate alla privacy: "ring signatures" (firme ad anello), "ring confidential transactions" (RingCT, transazioni confidenziali ad anello) e "stealth addresses" (indirizzi stealth). Queste tre tecnologie mascherano rispettivamente l'iniziatore della transazione, l'ammontare della transazione ed il destinatario della transazione. Tutte le transazioni sulla rete Monero sono private: non c'è alcuna possibilità di effettuare, neanche accidentalmente, una transazione trasparente. Questa caratteristica è tipica solo di Monero. Non hai bisogno di dare fiducia ad alcuno per mantenere la tua privacy.
```

Comment:
`l'iniziatore della transazione, l'ammontare della transazione ed il destinatario della transazione.` -> `il mittente, l'ammontare e il destinatario della transazione`

Reply:
Are you sure it is possible to use "mittente" as "issuer" of a transaction? I didn't get this precise translation on internet and I'm not expert on finance :)
Anyways I can switch to "mittente", it is clearer.

Reply:
Yes, we also used mittente in other translations